README.adoc

```diff
@@ -471,6 +471,79 @@ To remove the image, run the following command:
 docker rmi openliberty-getting-started:1.0-SNAPSHOT
 ```
 
+// =================================================================================================
+// Developing the application in a Docker container
+// =================================================================================================
+
+== Developing the application in a Docker container
+
+// File 0
+server.xml
+[source, xml, linenums, role='code_column']
+----
+include::finish/src/main/liberty/config/server.xml[]
+----
+
+The Open Liberty Maven plug-in includes a `devc` goal that simplifies developing
+your application in a Docker container. This goal builds a Docker
+image, mounts the required directories, binds the required ports, and then runs
+the application inside of a container. This development mode, known as dev mode, also listens for
+any changes in the application source code or configuration and rebuilds the
+image and restarts the container as necessary.
+
+Build and run the container by running the devc goal from the `start` directory:
+
+[role='command']
+```
+mvn liberty:devc
+```
+
+After you see the following message, your application server in dev mode is ready:
+
+[source, role="no_copy"]
+----
+************************************************************************
+*    Liberty is running in dev mode.
+----
+
+Open another command-line session and run the `docker ps` command to verify that your container started:
+[role='command']
+```
+docker ps
+```
+
+Make sure that your container is running and does not have `Exited` as its status:
```

Comment:
what should the user do if the status is "exited" ?

Reply:
I have replaced this sentence with `Your container should be running and have Up as its status:`. We don't usually present troubleshooting/error handling options in the guide, so I think this new sentence is more consistent with other guides.